packages/ui-library/src/components/select/index.tsx

```diff
@@ -179,11 +176,8 @@ function Select<T extends { key: string | number }, H = object>({
       }}
     >
       {!disabledSearch && (
-        <ListSubheader disableGutters>
+        <ListSubheader disableGutters sx={{ marginBottom: SPACING(2.5) }}>
           <TextField
-            size="small"
-            // Autofocus on textfield
-            autoFocus
```

Comment:
we definitely want this

Reply:
This was suggested by the designer in the notion, I'll undo it anyway!